app/views/devices_guidance/preparing_4g_wireless_routers.md

```diff
@@ -12,9 +12,10 @@ Your technical contact must also keep accurate records of who has each device, a
 
 ## Contract length
 
-The SIM card inside each 4G wireless router will be activated the day after you receive them. You should loan the routers to the children, young people and their parents and carers the following day at the earliest. The data allowance starts from this point.
+The SIM card inside each router is activated the day after it is delivered to a trust, local authority or school. You should loan the 4G wireless routers to the children, young people and their parents and carers the following day at the earliest.
+
+The SIM card in each router was initially set to provide data for a 6 month period, but this has been extended through the end of January 2021 for routers received during the 2020 summer term. We are working on keeping data connections active until the end of July 2021 and will update this page with any developments.
```

Comment:
```suggestion
Each SIM card was initially set to provide data for a 6 month period, but this has been extended through the end of January 2021 for routers received during the 2020 summer term. We are working on keeping data connections active until the end of July 2021 and will update this page with any developments.
```

Avoid repetition of "The SIM card in each router"

Reply:
Hi Paul, the copy changes have been through an extensive sign off loop so we can't change wording at this point.

Reply:
This doesn't change the meaning of the content, so shouldn't need further sign-off.

Reply:
This was written by one content designer, signed off by another and approved by a policy lead - even v minor changes are. And we keep all copy in master docs so if we tweak wording at this stage in Git, there's a good chance it will be overwritten with the next release.